content/en/continuous_integration/static_analysis/rules/_index.md

```diff
@@ -456,6 +827,401 @@ python_security_data:
+  - link: "/continuous_integration/static_analysis/rules/typescript-inclusive/declarations"
+    tag: "declarations"
+    text: "Check declaration names for wording issues"
+  - link: "/continuous_integration/static_analysis/rules/typescript-inclusive/formal-parameters"
+    tag: "formal-parameters"
+    text: "Check parameter names for wording issues"
+  - link: "/continuous_integration/static_analysis/rules/typescript-inclusive/identifiers"
+    tag: "identifiers"
+    text: "Check identifier names for wording issues"
+typescript_node_security_data:
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/argon2"
+    tag: "argon2"
+    text: "Use strong security mechanisms with argon2"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/avoid-crypto-rc4"
+    tag: "avoid-crypto-rc4"
+    text: "Avoid RC4"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/avoid-crypto-sha1"
+    tag: "avoid-crypto-sha1"
+    text: "Avoid SHA1 security protocol"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/avoid-des"
+    tag: "avoid-des"
+    text: "Avoid DES and 3DES"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/chmod-permissions"
+    tag: "chmod-permissions"
+    text: "Do not give 777 permissions to a file"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/command-injection"
+    tag: "command-injection"
+    text: "Avoid command injection"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/crypto-avoid-weak-hash"
+    tag: "crypto-avoid-weak-hash"
+    text: "Avoid weak hash algorithm from CryptoJS"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/detect-buffer-noassert"
+    tag: "detect-buffer-noassert"
+    text: "Avoid calls to 'buffer' with 'noAssert' flag set"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/detect-child-process"
+    tag: "detect-child-process"
+    text: "Avoid instances of 'child_process' & non-literal 'exec()'"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/detect-eval-with-expression"
+    tag: "detect-eval-with-expression"
+    text: "Avoid `eval` with expressions"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/detect-new-buffer"
+    tag: "detect-new-buffer"
+    text: "Avoid Buffer(argument) with non-literal values"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/detect-non-literal-fs-filename"
+    tag: "detect-non-literal-fs-filename"
+    text: "Avoid variables in 'fs' calls filename argument"
+  - link: "/continuous_integration/static_analysis/rules/typescript-node-security/detect-non-literal-regexp"
+    tag: "detect-non-literal-regexp"
+    text: "RegExp with non-literal values it might allow a DOS attack"
```

Comment:
```suggestion
    text: "Regular expressions with non-literal values might allow a DOS attack"
```

Reply:
Same.

Over the limit of 60 characters. I've updated to `Detects non-literal values in regular expressions`